### PR TITLE
fix(ui): Entire header actions dropdown replaced with upsell

### PR DESCRIPTION
**Pull request by taylangocmen**
Fixes: [WOR-1747](https://getsentry.atlassian.net/browse/WOR-1747)

### CODE REVIEW

File: static/app/views/organizationGroupDetails/actions/index.tsx [hunk 1]
```diff
@@ -398,80 +399,69 @@ class Actions extends Component<Props, State> {
...
-                        priority: 'danger',
-                        label: t('Delete and discard future events'),
-                        hidden: !hasAccess,
-                        onAction: () => this.openDiscardModal(),
-                      },
-                    ]}
-                  />
-                </GuideAnchor>
-              )}
-            </Feature>
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/actions/index.tsx:474]: Are we getting rid of the Feature?

Comment (taylangocmen) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: No, what do you mean?

Comment (evanpurkhiser) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: I don't see the hook anymore?

Comment (evanpurkhiser) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: Sorry i meant `Feature` component usage

**Review by scttcper — APPROVED** [review]
i think we'll have to re-implement the upsell

**Comment by taylangocmen** [comment]
> i think we'll have to re-implement the upsell

@evanpurkhiser Do you mean the upsell feature? I think it's ok to remove it from this button because it's going to be under a ellipsis dropdown menu (hidden in this case), most people won't see it anyways

**Comment by evanpurkhiser** [comment]
In that case we should remove all the related hook code and upsell stuff as well right?

**Comment by taylangocmen** [comment]
> i think we'll have to re-implement the upsell

I'll make a task to get a new design for a new upsell with the `open-in-discover`

**Pull request merged**